### DISCUSSION TITLE
Escaping { character

### DISCUSSION THREAD

**Discussion by joshuavachon25** [pngwn/MDsveX#373] [Q&A]
Hi! When I use { in the body of my .svx, cause I want to write it, I receive an error whenever I try to write it. I also tried with \{ and html entities, but still not working. 

Can someone help me?

**Comment by abjutus** [comment]
```js
<script>
  let leftCurlyBracket = '{'
</script>

{leftCurlyBracket}
```
Not elegant, but works ...
